doc/src/contribute/observability.md

```diff
@@ -153,8 +171,23 @@ Also notice `elapsed_milliseconds` which logs the duration of each span.
 {"v":0,"name":"sui","msg":"[PROCESS_CERT - END]","level":20,"hostname":"Evan-MLbook.lan","pid":51425,"time":"2022-03-08T22:48:11.248688Z","target":"sui_core::authority_server","line":67,"file":"sui_core/src/authority_server.rs","tx_digest":"t#d1385064287c2ad67e4019dd118d487a39ca91a40e0fd8e678dbc32e112a1493","elapsed_milliseconds":2}
 ```
 
+### Jaeger (seeing distributed traces)
+
+To see nested spans visualized with [Jaeger](https://www.jaegertracing.io), do the following:
+
+1. Run this to get a local Jaeger container: `docker run -d -p6831:6831/udp -p6832:6832/udp -p16686:16686 jaegertracing/all-in-one:latest`
+2. Run sui like this (trace enables the most detailed spans): `SUI_TRACING_ENABLE=1 RUST_LOG="info,sui_core=trace" ./sui start`
```

Comment:
You should be able to use 1. for each numbered step and let the numbers be dynamically generated. Try it in preview.

Reply:
True though in my experience it's also harmless, my editor just does it.  However it does work as you say.